src/rgw/rgw_acl_swift.cc

```diff
@@ -119,10 +119,17 @@ void RGWAccessControlPolicy_SWIFT::to_str(string& read, string& write)
     ACLGrant& grant = iter->second;
     int perm = grant.get_permission().get_permissions();
     rgw_user id;
+    string url_spec;
```

Comment:
@smithfarm 
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits